tests/unit/lib/Any_unittest.cc

```diff
@@ -507,6 +512,60 @@ TEST(Any, clone_array2d)
 	delete[] src;
 }
 
+TEST(Any, clone_array_sgobject)
+{
+	int src_len = 3;
+	CSGObject** src = new CSGObject*[src_len];
```

Comment:
if you make a test where the pointer type is `Object**`, this will go into the `std::copy` branch above